### PR TITLE
sweet-nova: unstable-2023-04-02 -> unstable-2023-09-30

### PR DESCRIPTION
**Pull request by dr460nf1r3**
## Description of changes

Updates `sweet-nova`. This contains changes to the Konsole theme.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (there are none)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/themes/sweet-nova/default.nix [hunk 1]
```diff
@@ -2,17 +2,17 @@
 , gitUpdater
 , lib
 , stdenvNoCC
+,
```
Comment (PedroHLC) [inline pkgs/data/themes/sweet-nova/default.nix:5]: Borked line

**Review by PedroHLC — CHANGES_REQUESTED** [review]

**Comment by AndersonTorres** [comment]
Fix the commit message. It should have the form `<pkg>:  <oldVersion> -> <newVersion>`

**Comment by dr460nf1r3** [comment]
Fixed, thanks guys :)

**Review by AndersonTorres — APPROVED** [review]
Ping me after 24h to merge

**Comment by fabianhjr** [comment]
Ping after 24 hrs @AndersonTorres :3

**Pull request merged**